app/models/patient.rb

```diff
@@ -368,6 +371,47 @@ def latest_transfer
     end
   }
 
+  # Gets the current date in the patient's timezone
+  def curr_date_in_timezone
+    Time.now.getlocal(address_timezone_offset)
+  end
+
+  # Checks is at the end of or past their monitoring period
+  def end_of_monitoring_period?
+    return false if continuous_exposure
```

Comment:
Currently this method is only called from a query that already filters out `continuous_exposure = true` so this check doesn't do anything. Understand if you want to keep it around though because this could be used other places

Reply:
Yeah, want to keep it around in case this is used elsewhere in the future.